sql/driver/conn.go

```diff
@@ -213,6 +228,32 @@ func (c *conn) Select(p *parser.Select, args []driver.Value) (*rows, error) {
 	return nil, fmt.Errorf("TODO(pmattis): unimplemented: %T %s", p, p)
 }
 
+func (c *conn) ShowColumns(p *parser.ShowColumns, args []driver.Value) (*rows, error) {
+	desc, err := c.getTableDesc(p.Name)
+	if err != nil {
+		return nil, err
+	}
+
+	schema := structured.TableSchemaFromDesc(*desc)
+
+	// TODO(pmattis): This output doesn't match up with MySQL. Should it?
```

Comment:
I'm sure someone somewhere depends on the format of this output, but unless all databases output the same thing, I wouldn't bother.